octavia_f5/tests/unit/controller/worker/test_l2_sync_manager.py

```diff
@@ -109,9 +131,154 @@ def test_ensure_l2_flow_exception(self, mock_get_network,
             self.assertEqual("Failed ensure_l2_flow for all vcmp devices of network_id=test-network-id",
                              e.args[0])
 
-        mock_l2_flow.assert_called_once_with(selfips=[MOCK_SELFIP],
-            store={'bigip': self.manager._bigips[0], 'network': mock_get_network.return_value,
-                   'subnet_id': MOCK_FIXED_IP.subnet_id})
+        mock_l2_flow.assert_called_once_with(data={
+            'test-guest-hostname': {
+                'selfips': [MOCK_SELFIP],
+                'store': {'bigip': self.manager._bigips[0],
+                          'network': mock_get_network.return_value,
+                          'subnet_id': MOCK_FIXED_IP.subnet_id}}})
         mock_vcmp_l2_flow.assert_called_once_with(
             store={'bigip': self.manager._vcmps[0], 'network': mock_get_network.return_value,
                    'bigip_guest_names': [MOCK_BIGIP_HOSTNAME]})
+
+    @mock.patch("octavia.network.drivers.noop_driver.driver.NoopManager"
+                ".get_subnet")
+    def test__do_ensure_l2_flow_nothing_exist_no_errors(self, mock_get_subnet):
+        mock_get_subnet.return_value = network_models.Subnet(
+            id='test-subnet-id', gateway_ip='1.2.3.1',
+            cidr='1.2.3.0/24', network_id='test-network-id')
+        mock_network = f5_network_models.Network(
+            mtu=9000, id='test-network-id', subnets=['test-subnet-id'],
```

Comment:
Probably cosmetic, but MTU standard in Converged Cloud is 8950

Reply:
It does not really matter here, it's just testing data and I can remove it completely